src/jmc/compile/lexer.py

```diff
@@ -31,11 +31,17 @@
     "worldgen/biome",
     "worldgen/configured_carver",
     "worldgen/configured_feature",
-    "worldgen/configured_structure_feature",
```

Comment:
JMC you still support the old version. So instead of changing the old one they both should be in there